avm/ptn/app-service-lza/hosting-environment/modules/supporting-services/deploy.supporting-services.bicep

```
@@ -0,0 +1,63 @@
+targetScope = 'resourceGroup'
+import { NamingOutput } from '../naming/naming.module.bicep'
+
+// ------------------
+//    PARAMETERS
+// ------------------
+
+param naming NamingOutput
+
+@description('The location where the resources will be created. This needs to be the same region as the spoke.')
+param location string = resourceGroup().location
+
+@description('Optional. The tags to be assigned to the created resources.')
+param tags object = {}
+
+@description('Required. Whether to enable deployment telemetry.')
+param enableTelemetry bool
+
+// Hub
+@description('The resource ID of the existing hub virtual network.')
+param hubVNetId string
```

```suggestion
param hubVNetResourceId string
```
[spec](https://azure.github.io/Azure-Verified-Modules/specs/bcp/res/#:~:text=ID%3A%20SNFR22%20%2D%20Category%3A%20Inputs%20%2D%20Parameters/Variables%20for%20Resource%20IDs)